src/SampSharp.GameMode/World/VehicleModelInfo.cs

```diff
@@ -285,14 +292,14 @@ private VehicleModelInfo(int type, string name, VehicleCategory category)
         /// </summary>
         /// <param name="vehicle">The vehicle to find information about.</param>
         /// <returns>An instance of <see cref="VehicleModelInfo" /> containing information about the specified vehicle.</returns>
-        public static VehicleModelInfo ForVehicle(BaseVehicle vehicle)
+        public static VehicleModelInfo ForVehicle(SampSharp.GameMode.World.BaseVehicle vehicle)
```

Comment:
Don't think the full namespace qualifier is required here?

Reply:
I think it's already fixed in the second commit.

Reply:
Ah yes